css/properties/position-area.json

```
@@ -21,7 +21,8 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": "131"
+              "version_added": "131",
+              "notes": "The property is parsed and accepted, although it does not have any effect as it also needs the implementation of <a href='https://developer.mozilla.org/docs/Web/CSS/anchor-name'>`anchor-name`</a> and <a href='https://developer.mozilla.org/docs/Web/CSS/position-anchor'>`position-anchor`</a>."
```

This sounds like a partial implementation?

Do I understand this right: The property only has an effect when _combined_ with anchor-name and (or?) position-anchor?

---

added

---

@dletorey Thanks for adding the partial option.

Could we make the note clearer? Something along these lines:

```suggestion
              "notes": "The property is parsed and accepted, but it has no effect alone, and both <a href='https://developer.mozilla.org/docs/Web/CSS/anchor-name'>`anchor-name`</a> and <a href='https://developer.mozilla.org/docs/Web/CSS/position-anchor'>`position-anchor`</a> are not yet supported."
```